functests/performance/performance.go

```diff
@@ -82,11 +82,11 @@ var _ = Describe("performance", func() {
 
 		// Check /usr/local/bin/pre-boot-tuning.sh existence under worker's rootfs
 		const perfRtKernelPrebootTuningScript = "/usr/local/bin/pre-boot-tuning.sh"
-		It(perfRtKernelPrebootTuningScript+" should exist on the nodes", func() {
+		It("[test_id:28523][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] /usr/local/bin/pre-boot-tuning.sh should exist on the nodes", func() {
 			checkFileExistence(workerRTNodes, perfRtKernelPrebootTuningScript)
 		})
 
-		It("Should inject systemd configuration files into initramfs", func() {
+		It("[test_id:28525][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] Should inject systemd configuration files into initramfs", func() {
```

Comment:
@gsr-shanks this was removed due to a revert patch we had to enforce. i will add this test again in a different PR with a stricter approach (to also test rpm-ostree latest fix)

Reply:
Ok, please use the same test_id.

Reply:
we only removed the reboot flag, so this test is fine, isn't it? It's green at least ;)

Reply:
test is fine. 
@dshchedr  once rpm-ostree fix confirmed (should be u/s already)  we can change the test to check setAffinity.conf and system.conf existence in lsinitrd twice (it more accurate)